src/doc/tutorials/MRI.rst

```diff
@@ -1,5 +1,3 @@
-.. _MRI:
-
 MRI
 ===
```

Comment:
Curious as to why you removed the tag?

Reply:
Because all top-level headings (and some nesting level of sub-headings) have auto-anchors created so it winds up being a dup and generates an error during build.

Reply:
Fair enough. But why not remove the tag from ccl.rst since you modified that file, too?

Reply:
Which tag are you referring to there? The one I removed from `MRI.rst` is because it was a dup. `.. _MRI:` is an exact match to the heading 

```
MRI
===
```

Sphinx automatically generates that anchor (your calling tag) and so explicitly adding one of *exactly* the same name creates an error. I don't see anything similar in `CCL.rst`.

Reply:
BTW...a *best practice* in Sphinx is **NOT** to use auto-anchors in cross-refs because if the heading being referenced is ever edited, the links to it break. So, it is indeed *better* to have an explicitly created link target/anchor.  Being lazy, I didn't wanna go find any/all refs to `MRI` and update them with whatever substitute name I might have introduced such `.. _MRI_top:` to correct. So, removal was most expedient option.

Reply:
BTW (#2) I forgot you are on SRE this week and probably would have preferred not being asked to review a PR too. If so, lemme know.

Reply:
Thanks for the explanation. All good.